index.d.ts

```diff
@@ -159,7 +162,7 @@ declare module "react-native-twilio-video-webrtc" {
     enableAudio?: boolean;
     enableVideo?: boolean;
     encodingParameters?: {
-      enableH264Codec: boolean;
+      enableH264Codec?: boolean;
```

Comment:
I see it set it to `false` in current implementation in [app's branch 5320-conditional-emdr](https://github.com/senseobservationsystems/goalie-2-mobile-app/blob/5320-conditional-emdr/app/twilio/sagas.ts#L180).
Any particular reason why we change it to make it optional here? Should we just remove setting it in app's end?

Reply:
>Any particular reason why we change it to make it optional here?

In the upstream, they've added `null` check, so it will be optional.

>I see it set it to false in current implementation in [app's branch 5320-conditional-emdr](https://github.com/senseobservationsystems/goalie-2-mobile-app/blob/5320-conditional-emdr/app/twilio/sagas.ts#L180).

Since it back to optional, we can remove it.

Reply:
> I see it set it to false in current implementation in [app's branch 5320-conditional-emdr](https://github.com/senseobservationsystems/goalie-2-mobile-app/blob/5320-conditional-emdr/app/twilio/sagas.ts#L180).

If you take a look again I think you'll see it's already gone.

> Any particular reason why we change it to make it optional here? Should we just remove setting it in app's end?

This is an example of a feature that we added ourselves (in the ND branch), that then got added separately upstream. In the upstream repo, this parameter [is optional](https://github.com/blackuy/react-native-twilio-video-webrtc/blob/aa3b9f8c1a011e1df349aa3bc0b53203bde061f7/index.d.ts#L131), so we've made the same change here.

Reply:
Cool thanks guys!